lib/kits/core-ui/components/list-publications/main-container.js

```diff
@@ -0,0 +1,94 @@
+import Mustache from 'mustache';
+import {request} from '../../../core';
+import {ComponentHelper, translate} from '../helpers';
+import './main-container.styl';
+
+const defaultTemplate = `\
+<div class="sgn__publications">
+    <ul class="sgn-publications-list-items-container">
+        {{#publications}}
+        <li class="sgn-publications-li">
+            <div class="publications__item" data-id="{{id}}">
+                <div class="sgn-publications-list-content-img">
+                    <img data-id="{{id}}" src="{{images.view}}" alt="{{label}}">
+                </div>
+                <div class="sgn-publications-list-content-text">
+                    <div class="sgn-publications-list-content-heading">
+                        <span>{{label}}</span>
+                    </div>
+                    <div class="sgn-publications-list-content-date">
+                        <span>{{date_from}}-{{date_till}}</span>
+                    </div>
+                </div>
+            </div>
+        </li>
+        {{/publications}}
+    </u;>
+</div>
+<div id="sgn-paged-publication-container"></div\
+`;
+
+const mainContainer = ({configs, template, el}) => {
+    const translations = {
+        localeCode: translate('locale_code'),
+        untilLabel: translate('paged_publication_viewer_until_label')
+    };
+
+    const fetchPublications = () => {
+        return new Promise((resolve) => {
+            request(
+                {
+                    url: `/v2/catalogs`,
+                    qs: {
+                        dealer_id: configs.dealerId,
```

Comment:
Where does this come from? Shouldn't it be businessId?

Reply:
It came from here: https://github.com/shopgun/shopgun-js-sdk/blob/e06e072ac2dadb93b5ecf6e7af1fc50a0b2af300/lib/kits/core-ui/list-publications.js#L28

Reply:
OK, let's update it there too then.